server/controllers/finance/reports/financial.employee.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @overview server/controllers/finance/reports/financial.employee.js
+ *
+ * @description
+ * This file contains code to create a PDF report for financial activities of an employee
+ *
+ * @requires Employee
+ * @requires ReportManager
+ */
+const _ = require('lodash');
+const ReportManager = require('../../../lib/ReportManager');
+
+const Employee = require('../../payroll/employees');
+const Creditors = require('../../finance/creditors');
+
+const TEMPLATE = './server/controllers/finance/reports/financial.employee.handlebars';
+
+const PDF_OPTIONS = {
+  filename : 'FORM.LABELS.FINANCIAL_STATUS',
+};
+
+/**
+ * @method build
+ *
+ * @description
+ * This method builds the report of financial activities of an Employee.
+ *
+ * GET /reports/finance/employeeStanding/:uuid
```

Comment:
This doesn't have a `:uuid`.  It passes the `:uuid` via a querystring `?employee_uuid=xxxx`.